dts/arm/raspberrypi/rpi_pico/rp2350_reset.h

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
.h files should be placed under include rather than under dts.

https://github.com/zephyrproject-rtos/zephyr/pull/77859/commits/6c7e64c93561eff5a8c75b59d9229f915c46dead